static/app/components/events/traceEventDataSection/index.tsx

```diff
@@ -81,13 +81,23 @@ function TraceEventDataSection({
   const {recentFirst, raw, activeDisplayOptions} = state;
 
   useEffect(() => {
-    if (raw || activeDisplayOptions.includes(DisplayOption.FULL_STACK_TRACE)) {
+    if (
+      raw ||
+      (!defaultStateProps.fullStackTrace &&
```

Comment:
Is the `fullStackTrace` option sometimes defaulted to true?

Reply:
yes..if there is no in-app frames